src/components/manage/Controlpanels/Users/RenderUsers.jsx

```diff
@@ -56,6 +66,40 @@ class RenderUsers extends Component {
     const [user, role] = value.split('.');
     this.props.updateUser(user, role);
   }
+
+  onEditUserSubmit(data, callback) {
+    // Do not handle groups and roles in this form
+    delete data.groups;
+    delete data.roles;
+    this.props.updateUserData(data.id, data);
+    this.setState({ user: {} });
+    this.props.listUsers();
+    return toast.success(
+      <Toast
+        success
+        title={this.props.intl.formatMessage(messages.success)}
```

Comment:
I tried to change the full name of existing users via-edit button and the Toast message returns "undefined" somehow while the rest functionality works fine.